src/main/java/Main.java

```diff
@@ -15,14 +15,14 @@ public static void main(String[] args) {
                 System.exit(0);
             }
             if (inputArgs.length == 1 && inputArgs[0].equals("ShowFigures")) {
-                for (Figure f: figueList) {
+                figureList.sort( (Figure a, Figure b) -> a.getSurfaceArea() < b.getSurfaceArea() ? -1 : 1);
+                for (Figure f: figureList) {
                     System.out.println(f.toString());
                 }
```

Comment:
```suggestion
                figureList.forEach(System.out::println);
```
Tak jeszcze można by ładniej zrobić